add IP addresses for EU

joebraha opened the pull request:
## Summary & Motivation
Updates our docs with current IP info for our EU region.

## How I Tested These Changes
Built docs locally:

![Screenshot 2025-12-17 at 1.53.23 PM.png](https://app.graphite.com/user-attachments/assets/98ebc1ab-9d61-4a60-a645-cdfdb2ec4a6a.png)

![Screenshot 2025-12-17 at 1.53.52 PM.png](https://app.graphite.com/user-attachments/assets/ef7d6d5b-7bc0-4fda-901d-a8a27def17ab.png)

## Changelog

> Insert changelog entry or delete this section.

neverett reviewed (approved):
thanks!

neverett reviewed (changes requested):
Actually, it looks like there are some formatting issues causing the docs build to fail -- can you run `yarn format` to fix them?

joebraha commented:
> Actually, it looks like there are some formatting issues causing the docs build to fail -- can you run `yarn format` to fix them?

fixed, and added `groupId` so that the two tabs in `management/dagster-ips.md` stay in sync

joebraha commented:
### Merge activity

* **Dec 19, 3:38 PM UTC**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.com/github/pr/dagster-io/dagster/33071).
* **Dec 19, 3:38 PM UTC**: @joebraha merged this pull request with [Graphite](https://app.graphite.com/github/pr/dagster-io/dagster/33071).
